src/Time/Units.hs

```diff
@@ -144,18 +142,18 @@ __Usage example:__
 "3h5m3s"
 
 -}
-type family (...) (a :: Rat) (b :: Rat) where
+type family (from :: Rat) ... (to :: Rat) where
     x ... y = TakeWhileNot y (DropWhileNot x AllTimes)
```

Comment:
Maybe variable names here can be changed from `x` and `y` to `from` and `to` as well.